tools/analyzer_plugin/lib/src/diagnostic/arrow_function_prop.dart

```diff
@@ -51,9 +51,6 @@ class ArrowFunctionPropCascadeDiagnostic extends ComponentUsageDiagnosticContrib
 
   @override
   computeErrorsForUsage(result, collector, usage) async {
-    // If there is only one cascaded prop, do not lint
-    if (usage.cascadedProps.length == 1) return;
-
```

Comment:
This is effectively already handled by the `allDescendantsOfType<CascadeExpression>(body).isEmpty` check below